queries.sql

```diff
@@ -46,5 +46,25 @@ SELECT COUNT(*) AS not_escaped FROM animals WHERE escape_attempts=0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT neutered, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY neutered;
-SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-1-1' AND date_of_birth <= '2000-12-1' GROUP BY neutered;
+SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-1-1' AND date_of_birth <= '2000-12-31' GROUP BY neutered;
+
+
+SELECT animals.name, owners.full_name FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE animals.owner_id = 4;
+
+SELECT animals.name, species.name FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name='Pokemon';
+
+SELECT animals.name, owners.full_name FROM animals LEFT JOIN owners ON animals.owner_id = owners.id;
+
+SELECT species.name, COUNT(animals.species_id) FROM animals JOIN species ON species.id = animals.species_id GROUP BY species.name;
+
+SELECT animals.name AS animal_name , owners.full_name AS owner_name FROM animals INNER JOIN owners ON owners.id = 2 AND species_id = 2;
+
+SELECT *  FROM animals WHERE owner_id = 5 AND escape_attempts = 0;
```

Comment:
- [ ] I think here you missed to make join to the owner table since your screenshot doesn't return anything but it sjould return the animal Charmander because it is own to Dean Winchester and has excape_attempts =0 so kindly update this to be correct 📌💪🏻💯🚀

Reply:
@Reem-lab Thank you.